src/core/flags.h.in

```diff
@@ -119,4 +134,25 @@
 #cmakedefine USE_ZLIB
 #endif
 
+/* functions to query flags at runtime instead of compile time */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int
```

Comment:
This style is totally wrong. Please take a look at existing files to get an actual idea, or read the (woefully outdated) CONTRIBUTING.md file. Naming scheme should likely be `mFlagGet`, and an enum of flags instead of individual functions would probably be better. Further, MGBA_EXPORT should be used.